bin/addCert.sh

```diff
@@ -3,4 +3,4 @@
 # Source env settings
 . .env
 
-ln -sf $(pwd)/services/storage/s04tls.crt ${CA_CERTS_TRUSTED_STORE}/s04.${LOCAL_DOMAIN}.tls.crt
+ln -sf "$(pwd)/services/storage/s04tls.crt" "${CA_CERTS_TRUSTED_STORE}/s04.${LOCAL_DOMAIN}.tls.crt"
```

Comment:
has it become more readable? or what is the reason for that change?

Reply:
Both `pwd` and `CA_CERTS_TRUSTED_STORE` may contain whitespace and it will break the script.

```
$ ls -l
total 0
-rw-rw-r-- 1 alexvanin alexvanin 0 Jun 15 18:33 a
$ pwd
/home/alexvanin/fun/ln/some dir

$ ln -s $(pwd)/a ./first
ln: target './first': No such file or directory
$ ln -s "$(pwd)/a" ./second

$ ls -l
total 0
-rw-rw-r-- 1 alexvanin alexvanin  0 Jun 15 18:33 a
lrwxrwxrwx 1 alexvanin alexvanin 33 Jun 15 18:37 second -> '/home/alexvanin/fun/ln/some dir/a'
```